Documentation/PullRequestConventions.md

```diff
@@ -0,0 +1,138 @@
+# Conventions for Pull Requests
+
+At present, the Babylon Native repository predominantly relies on an 
+"honor system" when it comes to pull requests, placing the bulk of the 
+responsibility for making sure the PR process remains effective and 
+efficient overwhelmingly in the hands of those submitting and performing 
+the code reviews. Consequently, this repo does not currently have strong 
+established practices for how pull requests, but we do have a number of 
+conventions which we typically try to follow when performing and receiving 
+code reviews. This document describes these conventions; it is formatted as 
+a series of  definitions for various actions and intended usages in a 
```

Comment:
A few spots with double spaces, "of  definitions" and "Babylon  Native" in the next line